package.json

```
@@ -16,7 +16,15 @@
     "type": "git",
     "url": "git+https://github.com/rapidsec-com/express-csp-generator.git"
   },
-  "keywords": [],
+  "keywords": [
```

Maybe add `content-security-policy`
```suggestion
  "keywords": [
      "content-security-policy",
```